scripts/pkgdep/mariner.sh

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env bash
+
+
+is_repo() { sudo tdnf repolist --all | grep -q "^$1"; }
+
+
+if [[ $ID == centos || $ID == rhel || $ID == rocky || $ID == mariner ]]; then
+	repos=() enable=("epel" "elrepo" "elrepo-testing") add=()
+	if ((${#add[@]} > 0)); then
+		for _repo in "${add[@]}"; do
+			sudo tdnf-config-manager --add-repo "$_repo"
+		done
+	fi
+
+	if ((${#repos[@]} > 0)); then
+		sudo tdnf install -y "${repos[@]}" sudo tdnf-utils
+		sudo tdnf-config-manager --enable "${enable[@]}"
+	fi
+fi
+sudo tdnf install -y build-essential
+sudo tdnf install -y CUnit-devel libaio-devel openssl-devel \
+	libuuid-devel libiscsi-devel ncurses-devel json-c-devel libcmocka-devel \
+	clang clang-devel python3-pip
+sudo tdnf install -y meson
+# Minimal install
+# workaround for arm: ninja fails with dep on skbuild python module
+if [ "$(uname -m)" = "aarch64" ]; then
+	pip3 install scikit-build
+	sudo tdnf install -y cmake
+	mkdir -p /tmp/bin/
+	ln -s /usr/bin/cmake /tmp/bin/cmake > /dev/null 2>&1 || true
+	export PATH=/tmp/bin:$PATH
+fi
+
+sudo tdnf install -y python python3-devel
+#Create hard link to use in SPDK as python
+if [[ ! -e /usr/bin/python && -e /etc/alternatives/python3 ]]; then
+	ln -s /etc/alternatives/python3 /usr/bin/python
```

Comment:
The comment appears misplaced.  "ln -s" is for soft link.

Reply:
I have removed the comment.